tests/cache/tests.py

```diff
@@ -1278,6 +1278,15 @@ def test_memcached_uses_highest_pickle_version(self):
     def test_memcached_options(self):
         self.assertEqual(cache._cache.server_max_value_length, 9999)
 
+    def test_set_many_returns_failing_keys(self):
+        def fail_set_multi(mapping, *args, **kwargs):
+            return mapping.keys()
+        with mock.patch('memcache.Client.set_multi', side_effect=fail_set_multi):
+            failing_keys = cache.set_many({
+                "key": "value",
```

Comment:
Use single quotes unless a string contains double quotes. Also, this looks fine to fine on the line above.